src/SpectralDataManager.h

```diff
@@ -0,0 +1,64 @@
+#include <tracks/playabletrack/wavetrack/ui/SpectrumView.h>
+#include <effects/Effect.h>
+#include "./SpectrumTransformer.h"
+
+class SpectralDataManager{
+public:
+   SpectralDataManager();
+   ~SpectralDataManager();
+   static int ProcessTracks(TrackList& tracks);
```

Comment:
So I understand this is the sole entry point to the code in this file, used by the dialog.  Separation of dialog code from the state it manages is a very good idea!

Maybe SpectralDataManager should be a namespace, and this the one function declared in it, and all the rest can be hidden in the corresponding .cpp file.